test/resque_inline_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+class IgnoreExceptionsTest < Minitest::Test
+  def setup
+    Resque.stubs(:redis)
+    Resque.inline = true
+  end
+
+  def teardown
+    Resque.inline = false
+    Resque.unstub(:redis)
+  end
+
+  def test_ignore_exceptions
+    GoodJob.stubs(:perform).raises(CustomException)
```

Comment:
Although this is valid, I'd rather see an existing job be re-used without the need to stub the method like this, we've not done this anywhere else in the code base so I'd like to see it consistent.